lib/pdf/inspector/text.rb

```diff
@@ -62,6 +64,11 @@ def set_word_spacing(*params)
         @word_spacing << params[0]
       end
       
+      def set_horizontal_text_scaling(*params)
```

Comment:
To be honest, for this, given that the PDF operators Tc (character spacing) and Tz (horizontal text scaling) seem to work very similarly, I've copied the method from set_character_spacing.